INSTALL

```diff
@@ -124,9 +124,9 @@ sudo yum install -y gcc gdb make git libtool autoconf automake byacc flex openss
 
 For SELinux support you will need selinux-policy-devel package and specify `--with-selinux-policy` to `autogen.sh` or `configure`
 
-* Debian (Debian 12 2023-10-09)
+* Debian (Debian 12 2024-06-21)
 
-sudo apt-get install -y build-essential git libtool autoconf automake bison flex libssl-dev libpcre2-dev libbison-dev libacl1 libacl1-dev lmdb-utils liblmdb-dev libpam0g-dev libtool libyaml-dev libxml2-dev
+sudo apt-get install -y build-essential git libtool autoconf automake bison flex libssl-dev libpcre3-dev libpcre3 libbison-dev libacl1 libacl1-dev lmdb-utils liblmdb-dev libpam0g-dev libtool libyaml-dev libxml2-dev
```

Comment:
When I take this change and use it on debian-12 I get problems with autogen.sh, no problem if I install libpcre2-dev as before.

```
checking for pcre2_compile_8 in -lpcre2-8... no
configure: error: Cannot find PCRE2
```

Reply:
I tried to use this and then build with source tar ball for 3.12.5 and it failed make

```
root@bookworm:~/cfengine-3.12.5# make
Making all in libcompat
  CC       generic_at.lo
  CC       getopt.lo
  CC       getopt1.lo
  CC       strlcat.lo
  CC       strlcpy.lo
  CC       memdup.lo
  CC       rpl_ctime.lo
  CC       strrstr.lo
  CCLD     libcompat.la
Making all in libutils
  CC       alloc.lo
  CC       array_map.lo
  CC       buffer.lo
  CC       cfversion.lo
  CC       cleanup.lo
  CC       csv_writer.lo
  CC       csv_parser.lo
  CC       encode.lo
  CC       file_lib.lo
  CC       hash.lo
In file included from hash.c:33:
./hash.h:62:28: error: unknown type name ‘RSA’
   62 | Hash *HashNewFromKey(const RSA *rsa, HashMethod method);
      |                            ^~~
hash.c:212:28: error: unknown type name ‘RSA’
  212 | Hash *HashNewFromKey(const RSA *rsa, HashMethod method)
      |                            ^~~
hash.c: In function ‘HashNewFromKey’:
hash.c:224:5: error: implicit declaration of function ‘RSA_get0_key’ [-Werror=implicit-function-declaration]
  224 |     RSA_get0_key(rsa, &n, &e, NULL);
      |     ^~~~~~~~~~~~
cc1: some warnings being treated as errors
make[3]: *** [Makefile:634: hash.lo] Error 1
make[2]: *** [Makefile:506: all] Error 2
make[1]: *** [Makefile:685: all-recursive] Error 1
make: *** [Makefile:591: all] Error 2
```

Reply:
> We can let this PR go and maybe make a CFE ticket for moving this information to a script which installs the dependencies based on detected platform.

Not sure how important this is. Seems like it would be more of a pain, than useful?